labop/labop.ttl

```diff
@@ -395,6 +451,32 @@ labop:fromSamples a owl:ObjectProperty ;
     rdfs:domain labop:SampleData ;
     rdfs:range labop:SampleCollection .
 
+labop:sampleDescriptions a owl:DatatypeProperty ;
```

Comment:
Should this be `owl:ObjectProperty` with `rdfs:range sbol:Implementation`?

Reply:
I believe not, because it will be an xarray of Implementation URIs.